xcode/Subconscious/Shared/Components/Deck/CardStackView.swift

```diff
@@ -1,8 +1,10 @@
 import SwiftUI
 
 struct CardView: View {
+    @Environment(\.colorScheme) private var colorScheme
+    
     var entry: CardModel
-    @Environment(\.colorScheme) var colorScheme
+    var onLink: (EntryLink) -> Void
```

Comment:
Add onLink callback. I think we may not have been handling in-card link taps before?

Reply:
Confirmed in playtesting. We were not handling in-card link taps before.